ush/config_defaults.sh

```diff
@@ -2116,7 +2116,7 @@ LBCS_ICS_ONLY="FALSE"
 DO_POST_SPINUP="FALSE"
 DO_POST_PROD="TRUE"
 DO_PARALLEL_PRDGEN="FALSE"
-DO_SAVE_INPUT="FALSE"
+DO_SAVE_INPUT="TRUE"
```

Comment:
@hu5970 so for retros, we can set DO_SAVE_INPUT="FALSE" to save some disk spaces?

Reply:
We should set default to DO_SAVE_INPUT="FALSE" as all retros and most of real-time does not need save INPUT.

Reply:
Anyone wants this needs a clear purpose why to save those files.

Reply:
Changed it back to "FALSE". Skipping this step in save_restart task would save both disk space and computer time.